### PR TITLE
MAINT: removed NRL affiliation

### PR DESCRIPTION
**Pull request by aburrell**
Removed the NRL affiliations from the webpage.  Let me know what you think of this approach.  We can fine-tune it here or in the next PR into develop (following this one).

### CODE REVIEW

**Review by kzawdie — APPROVED** [review]
I think that is a good way to do it. Looks good to me!

File: index.html [hunk 1]
```diff
@@ -49,11 +49,11 @@ <h2>Our People</h2>
 	        <h3>Directors</h3>
 	        <p>
 	            <ul>
-                    <li>Dr. Angeline G. Burrell, Naval Research Laboratory (<i>Exec. Dir.</i>)</li>
+                    <li>Dr. Angeline G. Burrell, Ionospheric Researcher (<i>Exec. Dir.</i>)</li>
 	                <li>Dr. John Coxon, University of Southampton</li>
 	                <li>Dr. Alexa Halford, NASA Goddard Space Flight Center</li>
-	                <li>Dr. McArthur Jones Jr., Naval Research Laboratory</li>
-	                <li>Dr. Kate Zawdie, Naval Research Laboratory</li>
+	                <li>Dr. McArthur Jones Jr., Atmospheric Researcher</li>
```
Comment (mjonesjr) [inline index.html:55]: Upper Atmospheric Researcher

**Review by mjonesjr — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (aburrell) [reply index.html:55]:
```suggestion
	                <li>Dr. McArthur Jones Jr., Upper Atmospheric Researcher</li>
```

File: index.html [hunk 2]
```diff
@@ -49,11 +49,11 @@ <h2>Our People</h2>
 	        <h3>Directors</h3>
 	        <p>
 	            <ul>
-                    <li>Dr. Angeline G. Burrell, Naval Research Laboratory (<i>Exec. Dir.</i>)</li>
+                    <li>Dr. Angeline G. Burrell, Ionospheric Researcher (<i>Exec. Dir.</i>)</li>
 	                <li>Dr. John Coxon, University of Southampton</li>
 	                <li>Dr. Alexa Halford, NASA Goddard Space Flight Center</li>
```
Comment (aburrell) [inline index.html:54]:
```suggestion
	                <li>Dr. John Coxon, University of Southampton, Magnetospheric Researcher</li>
	                <li>Dr. Alexa Halford, NASA Goddard Space Flight Center, Magnetospheric Researcher</li>
```

**Review by mjonesjr — APPROVED** [review]

**Pull request merged**